examples/Microsoft.DocumentDB_mongoClusters_firewallRules@2025-09-01/main.tf

```diff
@@ -0,0 +1,82 @@
+terraform {
+  required_providers {
+    azapi = {
+      source = "Azure/azapi"
+    }
+    time = {
```

Comment:
time provider seems not used, please remove it.

Reply:
fixed